pkgs/tools/text/runiq/default.nix

```diff
@@ -1,15 +1,21 @@
-{ lib, rustPlatform, fetchCrate, stdenv, darwin }:
+{ lib, rustPlatform, fetchFromGitHub, stdenv, darwin }:
 
 rustPlatform.buildRustPackage rec {
   pname = "runiq";
-  version = "2.0.0";
+  version = "2.0.0-unstable-2024-08-19";
 
-  src = fetchCrate {
-    inherit pname version;
-    hash = "sha256-qcgPuJOpK2fCsHAgzoIKF7upb9B3ySIZkpu9xf4JnCc=";
+  src = fetchFromGitHub {
+    owner = "whitfin";
+    repo = "runiq";
+    rev = "a642926f6ec09d4faeebebb563d4aed89e0d36fb";
+    hash = "sha256-DWP0kbTjXlyUI/+bHgom9/XJ2XW/BJEU4xvIisPVug0=";
   };
 
-  cargoHash = "sha256-WSMV0GNKNckN9uSPN647iDloGkNtaKcrZbeyglUappc=";
+  cargoLock.lockFile = ./Cargo.lock;
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
```

Comment:
Why do we need this, if the point of updating to an unstable version is that upstream has already fixed the problem?

Reply:
Because there is no Cargo.lock upstream. I think `cargo vendor` requires network access unless I am mistaken?